raiden/messages/metadata.py

```diff
@@ -79,6 +79,9 @@ class Metadata:
     # inititally create the Metadata object as part of a message - this is the case when we are
     # the initiator of a transfer.
     original_data: Optional[Any] = None
+    encrypted_secret: Optional[EncryptedSecret] = field(
+        default_factory=lambda: EncryptedSecret(b"")
+    )
```

Comment:
also here

Reply:
Maybe as another context. If we can't provide the encrypted secret (because of missing metadata) or we dont want to, then there should be no field in the metadata at all called "encrypted secret"